src/MobileUI/Resources/Styles/Templates.xaml

```diff
@@ -5,25 +5,28 @@
                     xmlns:resolver="clr-namespace:Maui.Plugins.PageResolver;assembly=Maui.Plugins.PageResolver"
                     xmlns:vm="clr-namespace:SSW.Rewards.Mobile.ViewModels">
     <ControlTemplate x:Key="PageTemplate">
-        <Grid RowDefinitions="55,*"
+        <Grid RowDefinitions="60,*"
               Margin="0"
               Padding="0"
-              TranslationY="{OnPlatform iOS=-5, Default=0}">
+              TranslationY="{OnPlatform iOS=-10, Default=0}">
             <Grid.BindingContext>
                 <resolver:ResolveViewModel x:TypeArguments="vm:TopBarViewModel" />
             </Grid.BindingContext>
-            <Grid ColumnDefinitions="60,*,60"
-                  Margin="0"
+            <Grid Grid.Row="0"
+                  ColumnDefinitions="60,*,60"
+                  HeightRequest="60"
+                  Padding="10,5"
                   BackgroundColor="{StaticResource Background}"
+                  VerticalOptions="Start"
                   x:DataType="vm:TopBarViewModel">
                 <!-- Wrapping AvatarView in a Grid where IsVisible can be toggled to work around 
```

Comment:
Can this comment be deleted? The linked issue is closed and marked as resolved.

Reply:
I think the issue still persists - I think that comment just describes the workaround that was implemented from that PBI.